content/FAQ.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Frequently Asked Questions
+url: FAQ/
+summary: Frequently asked questions
+showToc: true
+tocopen: true
+---
+
+## I read the docs but still need help!
+
+Come join the [discord](https://discord.gg/PeHR6Tjyvn)! Let the community help you in real-time!
+
+## What is VPupPr
+
+VPupPr is an open-source project that animates 3D objects using the glb or vrm file extension. 
+Most notable for using 3D vtube models as its main usage.
+
+## Platform Supported
+
+Currently supported on windows and common linux distributions. MacOS/OSX is not currently supported,
+feel free to fork the github [repo](https://github.com/virtual-puppet-project/vpuppr) and offer it to
```

Comment:
"offer it to the community" has a weird vibe to it. I would personally leave it out since most of this section refrains from commentary.